src/util/webauthn.js

```diff
@@ -0,0 +1,64 @@
+/* jshint es5:true */
```

Comment:
At the moment all logic specific for the [Webauth polyfill](fido-snippets/polyfill/webauthn.js) is removed.

All current logic in the file is based on the [MSCredentials object spec](https://msdn.microsoft.com/en-us/library/mt697639(v=vs.85).aspx).

I don't think `webauthn.js` file is third-party. It is our own util.
Will cover it with tests.